tests/contrib/mesh/test_converter.py

```diff
@@ -186,55 +221,74 @@ def test_converter_rename():
 
     expected = [
         {
-            "id": "https://id.nlm.nih.gov/mesh/D017394",
+            "id": "https://id.nlm.nih.gov/mesh/D044467",
             "type": "rename",
             "scheme": "MeSH",
-            "subject": "RNA, Guide",
-            "new_subject": "RNA, Guide, Kinetoplastida"
+            "subject": "American Native Continental Ancestry Group",
+            "new_subject": "American Indian or Alaska Native"
         },
     ]
     assert expected == rename_ops
 
 
 def test_converter_replace():
     """Test replace scenarios only (sort-of)."""
-    orig_subject_1, replacement_subject_1 = (
-        replaced_subjects_qualified_already_present()
+    subject_replace_present_src, subject_replace_present_dst = (
+        subjects_qualified_replaced_already_present()
+    )
+    subject_replace_add_src, subject_replace_add_dst = (
+        subjects_replaced_newly_added()
+    )
+    subject_rename_src, subject_rename_dst = subjects_qualified_renamed()
+    subject_replace_rename_src, subject_replace_rename_dst = (
+        subjects_replaced_renamed()
     )
-    orig_subject_2, replacement_subject_2 = replaced_subjects_newly_added()
     src = [
-        orig_subject_1,
+        subject_replace_present_src,
         subject_unqualified(),
-        replacement_subject_1,
-        orig_subject_2
+        subject_replace_present_dst,
+        subject_replace_add_src,
+        subject_replace_rename_src,
+        subject_rename_src,
     ]
     dst = [
         subject_unqualified(),
-        replacement_subject_1,
-        replacement_subject_2,
+        subject_replace_present_dst,
+        subject_replace_add_dst,
+        subject_replace_rename_dst,
+        subject_rename_dst,
```

Comment:
I am still not terribly happy with the clarity of the code here, but best I could come up with.

Reply:
I think this is more clear. Thank you.

Reply:
Eh, sometimes there's not much you can do 🤷 I think with a bit of reading someone can reasonably jump in and figure it out and that's about as good as it gets.